### DISCUSSION TITLE
Arbitrary peers behavior when using prefixes

### DISCUSSION THREAD

**Discussion by NakajimaTakuya** [tailwindlabs/tailwindcss#10311] [Help] [answered]
I find the peer- syntax difficult when using prefixes.
In particular, I was uncomfortable with the fact that is-foo requires a prefix, as in the following example.
Is this compiled result intended?

```html
<script src="https://cdn.tailwindcss.com"></script>
<script>
  tailwind.config = {
    prefix: 'u-'
  }
</script>
<div class="u-peer u-is-foo"></div>
<div class="peer-[.is-foo]:u-block hidden">aiueo</div>
```
https://codepen.io/nazomikan/pen/JjBJKam?editors=1010

**Comment by wongjn** [comment, marked as answer]
I don't think it is intended. This could be related to #10125.

**Reply by NakajimaTakuya** [reply]
Oh, thanks for letting me know!
I am so happy to hear that they are already merged.:heart:
